### DISCUSSION TITLE
Anzeige verwendet Action not State

### DISCUSSION THREAD

**Discussion by kevinhaybach** [Zefau/ioBroker.jarvis#2027] [Q&A]
Liegt der Fehler an meinen Einstellungen oder an v3.1.0-beta.11?
Ich habe einen IconButtonActionTaster angelegt aber er zeigt mir teils den State teils die Action an.
State schaltet auf 1 --> Anzeige wechselt auf True 
Taster wird betätigt --> Action wird True für 500ms anschließend False, damit wechselt die Anzeige auf auf False obwohl der State noch True ist

@Zefau Wie ist euer Vorgehen wenn ich denke, das es ein Fehler ist? Erst eine Diskussion starten oder direkt einen Issue öffnen?

![TasterTest](https://user-images.githubusercontent.com/83004345/199937651-390bb8e8-5ab7-4fdf-9a6b-636bb65a6e1a.gif)

``` 
    "name": "TasterTest",
    "icon": "",
    "label": "",
    "function": "light",
    "states": {
      "state": {
        "stateKey": "state",
        "showState": true,
        "state": "0_userdata.0.Test.DimmerState",
        "action": "0_userdata.0.Test.DimmerSwitch",
        "actionElement": "IconButtonAction",
        "IconButtonActionConfig": {
          "pushButton": true
        },
        "properties": {
          "initial": false,
          "setOn": true,
          "setOff": true
        }
      }
    },
    "options": {
      "suppressPopup": true
    },
    "attributes": {},
    "id": "6862f101-4a7e-4fa6-9a26-c6518d0a29e8",
    "revision": "5b64a"
  ```

**Comment by Zefau** [comment]
Was ist denn dein Ziel?

Der Taster schaltet auf den zuvor eingestellten Wert, sofern kein `initial` angegeben ist. Bei dir ist `initial` drin, daher springt er auf `false`.

**Reply by kevinhaybach** [reply]
Der Taster funktioniert wie er soll aber der Anzeigewert ändert sich obwohl der Anzeigedatenpunkt DimmerState sich nicht ändert.

**Reply by Zefau** [reply]
Wie gesagt, er springt auf den von dir definierten `initial` Wert:

![image](https://user-images.githubusercontent.com/14368811/200190087-a19f68bd-58b5-433e-bf46-df91eeb005c1.png)

**Reply by kevinhaybach** [reply]
Ich denke wir reden noch aneinander vorbei oder ich habe ein falsches Verständnis. Ich habe für state und action unterschiedliche Datenpunkte vergeben. Der Datenpunkt action verhält sich wie erwartet. Er springt für die Pulsdauer auf True und anschließend auf initial. Die Anzeige in meinem GIF links neben dem Button zeigt aber den Wet von Action und nicht state BISzu dem Zeitpunkt an dem der State in meinem Fall DimmerState wieder seinen Zustand ändert.

**Reply by Zefau** [reply]
ok verstehe

**Comment by Zefau** [comment]
https://github.com/Zefau/ioBroker.jarvis/issues/2035
